addons/mail/static/src/components/chatter/chatter_tests.js

```diff
@@ -13,6 +14,8 @@ const {
     start,
 } = require('mail/static/src/utils/test_utils.js');
 
+const FormView = require('web.FormView');
+
```

Comment:
I think I don't like this. If you need a full form view and not just a chatter for your test, this should be inside form rendered tests then.